common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsIdTokenClaims.java

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.identity.common.internal.providers.microsoft.microsoftsts;
+
+import com.microsoft.identity.common.internal.providers.microsoft.MicrosoftIdTokenClaims;
+
+/**
+ * IdToken claims emitted by the Microsoft STS (V2).
+ *
+ * @see <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-tokens">Azure Active Directory v2.0 tokens reference</a>
+ */
+public class MicrosoftStsIdTokenClaims extends MicrosoftIdTokenClaims {
+    /**
+     * The time at which the token becomes invalid, represented in epoch time. Your app should use
+     * this claim to verify the validity of the token lifetime.
+     */
+    public static final String EXPIRATION_TIME = "exp";
+
+    // TODO Could not locate documentation for the following fields.
```

Comment:
@danieldobalian Do you know where the following IDToken claims are documented? I couldn't track this information down

Reply:
MSAL, so I'm assuming v2: https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-tokens

Reply:
Yep.  That looks right